src/main/java/mods/railcraft/common/blocks/tracks/behaivor/CollisionHandler.java

```diff
@@ -70,6 +85,16 @@ private ItemStack getOveralls(Entity entity) {
             }
             return null;
         }
+        @Nullable
+        private ItemStack getRubberBoots(Entity entity) {
+            if (entity instanceof EntityPlayer) {
+                EntityPlayer player = ((EntityPlayer) entity);
+                ItemStack feet = player.getItemStackFromSlot(EntityEquipmentSlot.FEET);
+                if (feet != null && (ModItems.RUBBER_BOOTS.isEqual(feet, false, false) || ModItems.STATIC_BOOTS.isEqual(feet, false, false))) //&& !((EntityPlayer) entity).capabilities.isCreativeMode)
+                    return feet;
+            }
+            return null;
```

Comment:
Use InvTools.emptyStack() for all future ItemStack null returns please. Needed for 1.11.